api/services/policy/src/engine/entities/Policy.ts

```diff
@@ -57,7 +59,7 @@ export class Policy implements PolicyInterface {
       name: this.name,
       start_date: this.start_date,
       end_date: this.end_date,
-      handler: this.handler.constructor.name,
+      handler: (this.handler.constructor as PolicyHandlerStaticInterface).id,
```

Comment:
@nmrgt J'pense que ça n'avait pas d'impacte avant mais ici : 
`this.handler.constructor.name` retourne des valeurs comme '_a' est-ce que c'était un fonctionnement voulu ? 
J'ai modifié pour que ça retourne l'id du handler par exemple 'occitanie_2022'.

Reply:
yes je pense que c'est correct